### ISSUE TITLE
Beacon on window focus

### ISSUE THREAD

**Issue by Gastove** [Malabarba/beacon#37]
Caveat: it's possible beacon can already to what I want, and I just don't grok how. Maybe this is a feature request? Hard to say.

A time when beacon could improve my programming life dramatically would be on changing window focus. The scenario goes like so: I have emacs subdivided in to four or five windows; I change window focus to a browser to look up documentation, say; when I re-focus on emacs, I have to find point. 

So. Should `beacon-blink-when-window-changes` already be doing this? (Beacon fires for me when I change between windows _within emacs_, but not when I change between application windows.) Would this be possible at all? Another approach would be a `defun` to allow a "where is my cursor?" keyboard command -- thoughts?

Thanks for the awesome package!

**Comment by Malabarba** [comment]
> So. Should beacon-blink-when-window-changes already be doing this?

No, but there's a variable for this. It's called `beacon-blink-when-focused`, but your milleage may vary.

> Would this be possible at all? Another approach would be a defun to
> allow a "where is my cursor?" keyboard command -- thoughts?

Yes, there should be a command for this. That was supposed to be the beacon-blink command, but there's a bug which prevents it from working currently.

> Thanks for the awesome package!

You're welcome. :-)
